README.md

```diff
@@ -27,3 +28,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+### Invariant explanation
+The invarient that must be true at the beginning of every recursive call is that n must be greater than 1.
+This is because it keeps calling the function until n == 1 and that starts the array with [0, 1] and
+continues to add to that until it adds the n'th term to the end of the array. Otherwise, if the initial case
+is n == 1 or n == 0, there is no recursion necessary. This makes sure that we always have the correct n'th number because we will always have the correct starting numbers. The most important part of the fibonacci sequence is to have the 0th and 1st numbers because the rest of the series is based off of that.
```

Comment:
Ok, but what does that tell you about the *current* iteration?

Reply:
I think I have it fixed where in the current iteration, n>1 means that this iteration will result in another recursive call.